lib/task_listener.js

```diff
@@ -39,6 +30,12 @@ export default class TaskListener extends EventEmitter {
     this.runningTasks = [];
     this.taskQueue = new TaskQueue(this.runtime);
     this.taskPollInterval = this.runtime.taskQueue.pollInterval;
+
+    if (this.runtime.isolatedContainers) {
```

Comment:
Ah. We could. There were some code changes I was making when coming up with devicemanagement and didn't update this: it can fit in user data. Or at least it's working now for me